test/harness/testTypedArray.js

```diff
@@ -48,13 +30,3 @@ assert.sameValue(typedArrayConstructors[5], Uint32Array);
 assert.sameValue(typedArrayConstructors[6], Uint16Array);
 assert.sameValue(typedArrayConstructors[7], Uint8Array);
 assert.sameValue(typedArrayConstructors[8], Uint8ClampedArray);
-
-assert.sameValue(typedArrayConstructors[0], TAConstructors[0]);
-assert.sameValue(typedArrayConstructors[1], TAConstructors[1]);
-assert.sameValue(typedArrayConstructors[2], TAConstructors[2]);
-assert.sameValue(typedArrayConstructors[3], TAConstructors[3]);
-assert.sameValue(typedArrayConstructors[4], TAConstructors[4]);
-assert.sameValue(typedArrayConstructors[5], TAConstructors[5]);
-assert.sameValue(typedArrayConstructors[6], TAConstructors[6]);
-assert.sameValue(typedArrayConstructors[7], TAConstructors[7]);
-assert.sameValue(typedArrayConstructors[8], TAConstructors[8]);
```

Comment:
Lol... what the hell was the point of this? Did I do that? Gross.

Reply:
@rwaldron ![](https://akns-images.eonline.com/eol_images/Entire_Site/2016912/rs_634x1024-161012100536-rs_634x1024-140806173305-634.urkel.jpg)